deploy/templates/notificationtemplates/sandbox/userdeactivated/notification.html

```diff
@@ -41,23 +41,21 @@
 >
 
     <p>
-        You are receiving this email because you have a Developer Sandbox for Red Hat OpenShift
-        account associated with {{.UserEmail}}.
+        You are receiving this email because the Developer Sandbox account associated with {{.UserEmail}} is now deactivated.
+        All your data on Developer Sandbox has been deleted. You can start a new trial at {{.RegistrationURL}}, whenever you would like.
     </p>
 
     <p>
-        Your account is now deactivated and all your data on Developer Sandbox for Red Hat OpenShift has been deleted.
-        You can request new access by signing up again at {{.RegistrationURL}}
+        To learn more about how Red hat products, support and training could be benefit your organization <a href="https://www.redhat.com/en/contact">request a callback.</a>
     </p>
 
     <p>
-        Join the Dev Sandbox community to share your feedback, request extension for your Sandbox environment from the #dev-sandbox channel on DevNation slack workspace.
-        You can join using this <a href="https://join.slack.com/t/rhdevnation/shared_invite/zt-27vfuywt3-TC7QuckHfxHatLj6Abk2_A">invite</a>. You can also reach us via email at {{.ReplyTo}} with any questions.
+        To share feedback about your experience with the Developer Sandbox email us at {{.ReplyTo}}.
```

Comment:
TBH, I never really know how commas should be used properly in English :-)
```suggestion
        To share feedback about your experience with the Developer Sandbox, email us at {{.ReplyTo}}.
```

Reply:
done in [2c1b589](https://github.com/codeready-toolchain/host-operator/pull/1138/commits/2c1b589ecb53d69829547f70aaa76d06fff3b1f1)